homework/style.css

```diff
@@ -1,3 +1,68 @@
+*{
```

Comment:
On a mobile device long descriptions wrap to the extreme left end. This does not look very nice. Perhaps an HTML table would work better.

![image](https://user-images.githubusercontent.com/2788771/65255336-fac05300-dafd-11e9-9947-557346301b93.png)